lib/model/model.go

```diff
@@ -2516,6 +2568,14 @@ func (m *model) checkFolderRunningLocked(folder string) error {
 	return errFolderNotRunning
 }
 
+func (m *model) PendingDevices() (map[protocol.DeviceID]db.ObservedDevice, error) {
+	return m.db.PendingDevices()
+}
+
+func (m *model) PendingFolders(device protocol.DeviceID) (map[string]map[protocol.DeviceID]db.ObservedFolder, error) {
```

Comment:
Are you using `protocol.EmptyDeviceID` to implicate returning infos for all devices? At the very least this requires documentation. Personally I'd prefer `devices []protocol.DeviceID` and then `nil` means everything - seems less magic to me. However either way, main point being comments explaining the behaviour ;)

Reply:
Since this method just forwards 1:1 to the DB package function, do you think that it should be documented in both places?

Reply:
> Personally I'd prefer `devices []protocol.DeviceID` and then `nil` means everything

That would not translate well to the REST API, which is currently the only user.  What I tried to achieve with having this argument is to possibly give a list of pending folders in the GUI's edit device modal, so only a single device is relevant or all of them.

Reply:
If it's just a single device to filter, then a slice indeed doesn't make any sense. I am still not sure whether I like using an empty dev ID meaning "do-not-filter" or whether I'd prefer having two methods (e.g. `PendingFolders()` and `PendingFoldersForDevice(device)`), but that's deep into bikeshed territory so lets keep it as is and potentially revisit on a "final polish review round".